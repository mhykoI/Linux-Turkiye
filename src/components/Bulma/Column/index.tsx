import React from 'react';
import clsx from 'clsx';
import { Bulma } from '../../../models';

export interface ColumnProps {
  size?: Bulma.ColumnSize;
}

const Column: React.FC<ColumnProps> = ({ size, children }) => {
  return (
    <div className={clsx('column', { [`is-${size}`]: size })}>{children}</div>
  );
};

export default Column;
