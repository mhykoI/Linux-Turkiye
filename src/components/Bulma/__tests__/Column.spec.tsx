import React from 'react';
import renderer from 'react-test-renderer';
import Column from '../Column';

describe('<Column />', () => {
  test('should render correctly', () => {
    const tree = renderer
      .create(<Column size={6}>This is a column!</Column>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
