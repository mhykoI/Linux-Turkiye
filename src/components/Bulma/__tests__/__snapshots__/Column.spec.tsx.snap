// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Column /> should render correctly 1`] = `
<div
  className="column is-6"
>
  This is a column!
</div>
`;
